source/UIABrowseMode.py

```diff
@@ -325,10 +326,28 @@ def event_caret(self,obj,nextHandler):
 		pass
 
 	def _getTableCellAt(self,tableID,row,column):
-		# for UIA, tableID is actually a UIA NVDAObject
-		tableElement=tableID.UIAGridPattern
+		startUIAElement=self.selection.UIAElementAtStart
+		# Comtypes cannot automatically cast a tuple to a variant in method calls, so cast to a Python array first

```

Comment:
This comment is not quite correct. It casts a tuple to a safearray of variants, but you want a safearray of ints. You get that by casting to a Python int array first.